modules/message/state/data/MessageData.ts

```diff
@@ -6,4 +6,5 @@ export type MessageData = {
   readonly username?: string
   readonly avatar_url?: string
   readonly files?: readonly File[]
+  readonly url?: string
```

Comment:
Remove this, this type is meant to directly translate to what Discord wants in a request, and shouldn't reflect application code
```suggestion
```